relayer/src/error.rs

```diff
@@ -521,19 +521,20 @@ impl Error {
 
 impl GrpcStatusSubdetail {
     /// Check whether this gRPC error matches
-    /// - status: InvalidArgument
     /// - message: verification failed: ... failed packet acknowledgement verification for client: client state height < proof height ...
     pub fn is_client_state_height_too_low(&self) -> bool {
-        if self.status.code() != tonic::Code::InvalidArgument {
-            return false;
-        }
+        // Gaia v6.0.1 (SDK 0.44.5) returns code`InvalidArgument`, whereas gaia v6.0.4
+        // (SDK 0.44.6, and potentially others) returns code `Unknown`.
+        // Workaround by matching strictly on the status message.
+        // if self.status.code() != tonic::Code::InvalidArgument
```

Comment:
Is there any reason to leave this old logic in the code as a comment? Would the SDK potentially move back to returning the old error code?

Reply:
I think it's quite nice to leave some documentation as to why we are not checking the status code like we do in the other places. I would vote for keeping it in.